ask-sdk-core/src/com/amazon/ask/response/ResponseBuilder.java

```diff
@@ -22,6 +26,8 @@
 import com.amazon.ask.model.dialog.DelegateDirective;
 import com.amazon.ask.model.dialog.ElicitSlotDirective;
 import com.amazon.ask.model.interfaces.audioplayer.AudioItem;
+import com.amazon.ask.model.interfaces.audioplayer.AudioItem.Builder;
```

Comment:
Nit: Is this reference needed?

Reply:
You are totally right. Reference to AudioItem builder was refactored so I could get rid of that import